go/private/actions/stdlib.bzl

```diff
@@ -82,6 +82,7 @@ def _build_stdlib_list_json(go):
         executable = go.toolchain._builder,
         arguments = [args],
         env = _build_env(go),
+        toolchain = "@io_bazel_rules_go//go:toolchain",
```

Comment:
I tried loading and using https://github.com/bazelbuild/rules_go/blob/master/go/private/go_toolchain.bzl#L26 instead of hardcoded string, but that creates a cycle:
 `//go/private:go_toolchain.bzl` ->  `//go/private/actions:stdlib.bzl` -> `//go/private:go_toolchain.bzl`
 I can break the cycle by moving `GO_TOOLCHAIN` to another file. WDYT?

Reply:
Sounds reasonable, please go ahead!

Reply:
I've moved it inside `go/private:common.bzl` and updated it inside files where its been used. Can you have another look? Thanks!

Reply:
Thanks!